src/vcpkg/spdx-exceptions.inc

```diff
@@ -1,45 +1,43 @@
-// Data downloaded from https://raw.githubusercontent.com/spdx/license-list-data/0af22f869f8b0906097cfac90ee0516992e8939f/json/exceptions.json
```

Comment:
Could the powershell script be modified to always produce consistent line endings? Preferably `\r\n` since that's what we already have.

Reply:
It is consistent, it was originally \n; the file was modified by removing the four spaces in front, which results in this diff.